src/Http/Http.Abstractions/test/RouteHandlerInvocationContextOfTTests.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Http.Abstractions.Tests;
+
+public class RouteHandlerInvocationContextOfTTests
```

Comment:
Might be a good idea to add a test to make sure the generated code is updated. 
Eg.: https://github.com/dotnet/aspnetcore/blob/main/src/Http/Http.Results/test/ResultsCacheTests.cs

Reply:
Good idea! Added a test for this.